mypy/main.py

```diff
@@ -459,6 +459,10 @@ def add_invertible_flag(flag: str,
     imports_group.add_argument(
         '--follow-imports', choices=['normal', 'silent', 'skip', 'error'],
         default='normal', help="How to treat imports (default normal)")
+    add_invertible_flag(
+        '--no-follow-imports-for-stubs', dest='follow_imports_for_stubs', default=True,
```

Comment:
Could you please add a test for this? (Probably in `check-flags.test`)

Reply:
Sorry I have no idea how to do it. :(
The tests file looks too cryptic to me.

Reply:
I performed basic manual functional checks, it looks like it works as it should.

Reply:
> I performed basic manual functional checks, it looks like it works as it should.

I trust you did this, but the test case is still needed. It is not that hard, try making something like `[case testNoImplicitReexport]` in that file.

Reply:
The problem is not adding a section, the problem is finding out the proper contents of it.
There's a ton of tests for ``--follow-imports`` there, so the proper content seems like not something trivial.

Unfortunately, I don't feel that I have the resources to dive into this cryptic test engine.

I didn't expect this PR would get this complex, I thought it would be limited to a simple formal change.

Reply:
We can just leave this issue and PR open until someone qulified finds time to complete it.

Or we can close it as WONTFIX.